mypy/checker.py

```diff
@@ -1886,6 +1886,10 @@ class C(B, A[int]): ...  # this is unsafe because...
                 ok = is_subtype(first_sig, second_sig, ignore_pos_arg_names=True)
         elif first_type and second_type:
             ok = is_equivalent(first_type, second_type)
+            if not ok and name in base2.abstract_attributes:
```

Comment:
Maybe I am missing something but I think this should be OK even if `name` is not abstract, so I would remove the right part of `and`.

Reply:
I think you're right; I had considered only the abstract case. Fixed.